src/tooltip/tooltip.ts

```diff
@@ -25,13 +25,17 @@ import {NgbTooltipConfig} from './tooltip-config';
 @Component({
   selector: 'ngb-tooltip-window',
   changeDetection: ChangeDetectionStrategy.OnPush,
-  host: {'[class]': '"tooltip show tooltip-" + placement', 'role': 'tooltip'},
+  host: {
+    '[class]': '"tooltip show tooltip-" + placement + (customClasses ? " " + customClasses : "")',
```

Comment:
I tried to use them but I had lot of issues trying to make it work. Could you please show me how should I use ES6 string template in the Component host? 

I tried many different things, including: `tooltip show tooltip-${placement} ${customClasses}` but it's not working (something like: cannot find placement...).